core/common/src/main/java/alluxio/Constants.java

```diff
@@ -62,6 +62,8 @@
   public static final String SCHEME_FT = "alluxio-ft";
   public static final String HEADER_FT = SCHEME_FT + "://";
 
+  public static final String HEADER_OBS = "obs://";
```

Comment:
"s3n" has been moved to the extension repo. do you want to have "obs" here?

Reply:
removed